src/core/mapping/core_mapper.cc

```diff
@@ -19,17 +19,22 @@
 #include "legate.h"
 
 #include "core/mapping/core_mapper.h"
+#include "core/mapping/machine.h"
+#include "core/mapping/operation.h"
 #ifdef LEGATE_USE_CUDA
 #include "core/comm/comm_nccl.h"
 #endif
 #include "core/task/task.h"
 #include "core/utilities/linearize.h"
 
-namespace legate {
+using LegionTask    = Legion::Task;
```

Comment:
What do the type aliases help here? It looks like it is just avoiding a couple `::` : )

Reply:
I believe this the type aliases are now gone.